packages/cli-hooks/src/get-manifest.js

```diff
@@ -38,9 +43,12 @@ function readManifestJSONFile(searchDir, filename) {
     if (jsonFilePath && fs.existsSync(jsonFilePath)) {
       return JSON.parse(fs.readFileSync(jsonFilePath, 'utf8'));
     }
-  } catch (error) {
-    console.error(error); // eslint-disable-line no-console
-    throw new Error('Failed to parse the manifest file for this project');
+  } catch (err) {
+    let message = 'Failed to parse the manifest file for this project';
+    if (err instanceof Error) {
+      message += `\n${err.name}: ${err.message}`;
+    }
+    throw new Error(message);
```

Comment:
Nicely done wrapping this error.

Not directly related to this PR or your work, and I think this issue has existed before, but it seems like in the case a hook returns a non-zero code to the CLI, the CLI seems to lose spacing of the output, for some reason?

Here is a screenshot of directly invoking the cli-hooks script from a project with a malformed manifest (left) vs. doing so w/ the CLI (right):

<img width="962" alt="Screenshot 2024-01-26 at 11 14 17 AM" src="https://github.com/slackapi/node-slack-sdk/assets/52645/16df471c-8cf3-46cd-9dd7-3eb9e601c38e">

Maybe just something to file over on the CLI as a future fix.

Reply:
Thanks! And that formatted error would be so much cleaner, good catch! I'll look into this soon but will add it to our backlog for now 🗄️